docs/source/index.mdx

```diff
@@ -12,63 +12,34 @@ specific language governing permissions and limitations under the License.
-
-| Accelerator                                                                                                            | Installation                                      |
-|:-----------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------|
-| [ONNX runtime](https://onnxruntime.ai/docs/)                                                                           | `python -m pip install optimum[onnxruntime]`      |
-| [Intel Neural Compressor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/neural-compressor.html)       | `python -m pip install optimum[intel]`            |
-| [Graphcore IPU](https://www.graphcore.ai/products/ipu)                                                                 | `python -m pip install optimum[graphcore]`        |
-| [Habana Gaudi Processor (HPU)](https://habana.ai/training/)                                                            | `python -m pip install optimum[habana]`           |
-
-
-If you'd like to play with the examples or need the bleeding edge of the code and can't wait for a new release, you can install the base library from source as follows:
-
-```bash
-python -m pip install git+https://github.com/huggingface/optimum.git
-```
-
-For the accelerator-specific features, you can install them by appending `#egg=optimum[accelerator_type]` to the `pip` command, e.g.
-
-```bash
-python -m pip install git+https://github.com/huggingface/optimum.git#egg=optimum[onnxruntime]
-```
+🤗 Optimum is an extension of 🤗 Transformers that provides a set of performance optimization tools to train and run models on targeted hardware with maximum efficiency.
+
+The AI ecosystem evolves quickly, and more and more specialized hardware along with their own optimizations are emerging every day.
+As such, 🤗 Optimum enables developers to efficiently use any of these platforms with the same ease inherent to 🤗 Transformers.
+
+🤗 Optimum is distributed as a collection of packages - check out the links below for an in-depth look at each one.
+
+<div class="mt-10">
+  <div class="w-full flex flex-col space-y-4 md:space-y-0 md:grid md:grid-cols-3 md:gap-y-4 md:gap-x-5">
+    <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/optimum/graphcore_index"
+      ><div class="w-full text-center bg-gradient-to-br from-blue-400 to-blue-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Optimum Graphcore</div>
+      <p class="text-gray-700">Train transformers on <a href="https://www.graphcore.ai/products/ipu">Graphcore IPUs</a>, a completely new kind of massively parallel processor to accelerate machine intelligence.</p>
+    </a>
+    <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/optimum/habana_index"
```

Comment:
```suggestion
    <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="./habana_index"
```

Reply:
please feel free to apply this suggestion to other ones, such as graphcore

Reply:
nice, thanks for the tip!!